modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_cp_definition_option_value_rel.jspf

```diff
@@ -227,124 +227,25 @@ long cpDefinitionOptionValueRelId = cpDefinitionOptionValueRelDisplayContext.get
 </c:if>
 
 <c:if test="<%= (cpDefinitionOptionValueRel == null) && !cpDefinitionOptionValueRelDisplayContext.isCPDefinitionOptionRelSelectDate() %>">
-	<aui:script require="frontend-js-web/index as frontendJsWeb, commerce-frontend-js/utilities/slugify as slugify">
-		var {debounce} = frontendJsWeb;
-
-		var form = document.getElementById(
-			'<portlet:namespace />cpDefinitionOptionValueRelfm'
-		);
-
-		var keyInput = form.querySelector('#<portlet:namespace />key');
-		var nameInput = form.querySelector('#<portlet:namespace />optionValueName');
-
-		if (nameInput) {
-			var handleOnNameInput = function () {
-				keyInput.value = slugify.default(nameInput.value);
-			};
-
-			nameInput.addEventListener('input', debounce(handleOnNameInput, 200));
-		}
-	</aui:script>
+	<liferay-frontend:components
```

Comment:
ehy @bryceosterhaus I was testing this PR locally and I found some issues that make us think it's safer for everybody to merge your changes after our PRs.
See:
I found this issue with this change of yours:

<img width="1239" alt="Screenshot 2024-04-04 at 10 40 40" src="https://github.com/liferay-commerce/liferay-portal/assets/580733/4cea9562-9ab3-401b-9b83-a337b52bfcbe">

this I think should be 
```suggestion
<liferay-frontend:component
```
and even with the suggested change I found this issue:

<img width="1676" alt="Screenshot 2024-04-04 at 10 44 00" src="https://github.com/liferay-commerce/liferay-portal/assets/580733/40c9c244-2a9f-41c3-b973-8a99bcfd56bc">

this is not working and we have no bandwidth to fix this issues too.